frontend/frontend-filters-links.php

```diff
@@ -388,7 +388,7 @@ public function check_canonical_url( $requested_url = '', $do_redirect = true )
 
 		// The language is not correctly set so let's redirect to the correct url for this object
 		if ( $do_redirect && $redirect_url && $requested_url != $redirect_url ) {
-			wp_redirect( $redirect_url, 301 );
+			wp_safe_redirect( $redirect_url, 301 );
```

Comment:
This one should have a phpcs ignore comment too and be changed back - same thing as the other one, sorry!